.eslintrc.js

```diff
@@ -1,15 +1,17 @@
 module.exports = {
-  parser: '@babel/eslint-parser',
-  plugins: ['jsdoc'],
+  parser: '@typescript-eslint/parser',
+  plugins: ['jsdoc', '@typescript-eslint'],
```

Comment:
What is this for?

Reply:
This provides more lint rules for .ts modules